src/components/SearchableSelect/SearchableSelect.tsx

```diff
@@ -438,8 +507,7 @@ const SearchableSelect = createClass({
 					)}
 					isDisabled={isDisabled}
 					onSelect={onSelect}
-					selectedIndices={isItemSelected ? [selectedIndex] : []}
-
+					selectedIndices={_.isNumber(selectedIndex) ? [selectedIndex] : []}
```

Comment:
Couldn't this remain checking `isItemSelected`?

Reply:
with just checking `isItemSelected`, typescript complains that `selectedIndex` may be null, but `selectedIndices` should be `number[]`.